<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>1251</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom du produit:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Seuil d alerte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 143, 62, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_6">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>240</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date sortie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_7">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>310</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prixunitaire</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>220</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
    <property name="placeholderText">
     <string>Donner le nom a rechercher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantite</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date entree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 143, 62, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Retourner vers le main</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
